### PR TITLE
Interaction States - NxTag - RSC-1481

### PR DESCRIPTION
**Pull request by EmilyBermudez88**
https://issues.sonatype.org/browse/RSC-1481

### CODE REVIEW

File: gallery/visualtests/nxTag.js [hunk 1]
```diff
@@ -46,17 +49,22 @@ describe('NxTag', function() {
...
+    it('has a blue outline and dark grey border when focused and hovered',
+        focusAndHoverTest(selector, undefined, OUTSET));
 
-    it('has a dark blue/grey background when clicked', async function() {
-      const [targetElement] = await waitAndGetElements(selector);
+    it('has a darker background and no border when selected', async function() {
+      const [targetElement, lastElement] = await waitAndGetElements(selector, lastTagSelector);
 
       await targetElement.click();
+      //remove focus from tag for snapshot
```
Comment (EmilyBermudez88) [inline gallery/visualtests/nxTag.js:65]: I couldn't figure out how to blur the element here - I tried blurring NxTag and the icon inside, but both left the focus outline in the snapshot. Any other ideas to remove the border are appreciated!

Comment (rpokorny) [reply gallery/visualtests/nxTag.js:65]: > I couldn't figure out how to blur the element here - I tried blurring NxTag and the icon inside, but both left the focus outline in the snapshot. Any other ideas to remove the border are appreciated!

NxSelectableTag is built around a checkbox input.  That's what you'd need to blur

Comment (EmilyBermudez88) [reply gallery/visualtests/nxTag.js:65]: > > I couldn't figure out how to blur the element here - I tried blurring NxTag and the icon inside, but both left the focus outline in the snapshot. Any other ideas to remove the border are appreciated!
> 
> NxSelectableTag is built around a checkbox input. That's what you'd need to blur

Ah of course, that makes sense. Thanks for the suggestion/feedback!

**Comment by entroform** [comment]
It looks good, but I did notice that we have the same issue in Safari:
I wonder if we should go back to using the position absolute solution 🤔 

Screenshot 2023-03-13 at 4 30 51 PM

**Comment by EmilyBermudez88** [comment]
>It looks good, but I did notice that we have the same issue in Safari:
I wonder if we should go back to using the position absolute solution 🤔

Makes sense to me! Back to `::before` here: [3be9e74](https://github.com/sonatype/sonatype-react-shared-components/pull/1009/commits/3be9e74fee37f20ef4b2d8eb5e39d2455122e95b)

**Review by entroform — APPROVED** [review]
🚀

**Pull request merged**